### ISSUE TITLE
NEW ISSUES SHOULD BE CREATED ON TRAC

### ISSUE THREAD

**Issue by pattonwebz** [WordPress/twentytwenty#991]
From this point on any new issues should be opened on the core trac instead of at GitHub to save them needing to be transitioned over there manually at a later point.

Please prefix tickets with `Twenty Twenty:` so that they are easy to find over there.

https://core.trac.wordpress.org

**Comment by ottok** [comment]
Why do you link to trac main page here? Please link to the list of twentytwenty issues directly. Thanks!

The commit https://github.com/WordPress/twentytwenty/commit/1025b5284f2927ed0b92f964b30243eab4d3b5e8#diff-3d9639f99b2cd6bda90b892bea090bef broke this plugin for PHP composer users, as there is a dotfile now shipped with the plugin and composer does not handle well those cases. This is a similar bug as when akismet started shipping a .htaccess file in May 2017.

```
# composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 4 installs, 0 updates, 0 removals
  - Installing johnpbloch/wordpress-core (5.3.0): Loading from cache
  - Installing wpackagist-plugin/auto-post-thumbnail (3.6.0): Downloading (100%)         
  - Installing wpackagist-plugin/google-analytics-dashboard-for-wp (5.3.9): Downloading (100%)         
  - Installing wpackagist-theme/twentynineteen (1.4): Downloading (100%)         
Generating autoload files
> WordPress\Installer::symlinkWPContent
Script WordPress\Installer::symlinkWPContent handling the post-install-cmd event terminated with an exception

                                                                                               
  [ErrorException]                                                                             
  rmdir(/data/wordpress/htdocs/wordpress/wp-content/themes/twentytwenty): Directory not empty                                                                                            
```

**Comment by pattonwebz** [comment]
Hey @ottok, we can certainly update the issue template so it links directly to the tickets, do you think also linking directly to the new ticket page would be good too?

As for composer problems with dotfiles in the directory - that is unfortunate and I will look into that and see what can be done, thank you for reporting.

**Comment by ottok** [comment]
No, just link to the issue overview. You want users to skim existing issues before going to file new ones.

For the composer issue I wrote a local fix to our copy of the general WordPress/Install.php used in in many gitified WordPress templates: https://github.com/Seravo/wordpress/commit/a42412cdb69074ab3b8645df6150adfae4cdd0c0
